samples/wifi/radio_test/sample_description.rst

```diff
@@ -128,9 +137,9 @@ Testing
 
               wifi_radio_test rx 0
               wifi_radio_test tx 0
+              wifi_radio_test init 1
```

Comment:
Can we use a channel other than 1, which makes this look like an enable flag.

Reply:
Changed to 11.